pkg/policy/api/rule_validation.go

```diff
@@ -382,20 +382,32 @@ func (pr *PortRule) sanitize(ingress bool) error {
 	}
 
 	listener := pr.Listener
-	if listener != "" {
+	if listener != nil {
```

Comment:
We could validate that if `listener.EnvoyConfig.Namespace` is empty for CECs (should only be specified for CCEC).